release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -524,6 +524,13 @@ With this release, you can install a cluster on Nutanix by using the named, prel
 
 For more information, see xref:../installing/installing_nutanix/installation-config-parameters-nutanix.adoc#installation-configuration-parameters-additional-nutanix_installation-config-parameters-nutanix[Additional Nutanix configuration parameters].
 
+[id="ocp-release-notes-installation-openstack-ipv6-single-stack-deployment_{context}"]
+==== Single-stack IPv6 clusters on {rh-openstack}
```

Comment:
I don't know if we want to "extend" the support to {rh-openstack} and {rhoso} or it can complicate things. The feature is supported in both versions of openstack.

Reply:
@rlobillo reviewed https://github.com/openshift/openshift-docs/pull/85554. 

Do you have any concerns about including RHOSO in this note?

Reply:
Hi @maxwelldb 

I believe that we are going to support 4.18 on top of RHOSO (which is just a new version for openstack). That will be reflected on the supportability matrix. I think we can keep using openstack here in a generic way and let the supportability matrix to reflect exactly which OCP release are supported on top of which OSP release.

Reply:
Fine with me!